src/idm/index.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import group from './group';
+import user from './user';
+import entity from './entity';
+import authentication from './authentication';
+
+
+import { errorHandler } from '../utils';
+
+const idm = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @namespace group
+    * @memberof agile.idm
+    **/
+    group: group(base,token),
+    /**
+    * @namespace user
+    * @memberof agile.idm
+    **/
+    user: user(base,token),
+    /**
+    * @namespace entity
+    * @memberof agile.idm
+    **/
+    entity: entity(base,token),
+    /**
+    * @namespace authentication
```

Comment:
So one thing I expect from the security implementation is to keep a global token in agile-sdk state. For instance we could init like `agile({token: adsfasd, url: resin.local }) and then it'd store the token with the closure like we do for the base url. Is that planned for later when the agile-api supports this?